src/utils/workspace.ts

```diff
@@ -32,3 +35,15 @@ export function isFolderOpenInWorkspace(fsPath: string): boolean {
         return false;
     }
 }
+
+export function getProjectType(projectPath: string): string {
+    let language: string = TemplateLanguage.JavaScript;
+    fse.readdirSync(projectPath).forEach((file: string) => {
```

Comment:
You should use async methods wherever possible (in this case, you would use `readdir` instead of `readdirsync`

Reply:
Using ```fse.pathExists()```.